client/src/actions/course.ts

```diff
@@ -66,6 +67,38 @@ export function loadCourse(id: number) {
     };
 }
 
+export function createCourse() {
+    return (dispatch: any) => {
+        dispatch(loading('create_course_success'));
+        fetch(`/api/course`, {
+            method: 'POST',
+            headers:  {
+                'Accept': 'application/json',
+                'Content-Type': 'application/json'
+            },
+            credentials: 'same-origin',
+            body: JSON.stringify({
+                name
```

Comment:
Where does name come from? If we don't have a way to input name on creation, #260 will become higher priority since all new courses will be names 'Untitled'.